movies_test.py

```diff
@@ -0,0 +1,111 @@
+"""Tests the functions of the code.py file"""
+
+import movies
+
+CUSTOMERS = {
+    "Bob" : {"movies":[],
+             "cash" : 100.0},
+    "Jim" : {"movies": ["Xmen 8: The Xmennening"],
+             "cash" : 10.0},
+    "Cary" : {"movies": ["Gigli: The Play: The Book: The movie", "The Bromance"],
+              "cash" : 120.0},
+    "Ricci": {"movies": [],
+              "cash" : 4.0}
+}
+MOVIE_TICKETS = {
+    "Xmen" : {"title" : "Xmen 8: The Xmennening", "seats" : 10},
+    "Bromance" : {"title" : "The Bromance", "seats" : 20},
+    "Gigli" : {"title" : "Gigli: The Play: The Book: The movie", "seats" : 102}
+    }
+def test_enough_money():
+    """Tests the enough money function"""
+    bob = {"movies":[], "cash" : 100.0}
+    ricci = {"movies": [], "cash" : 4.0}
+    assert movies.enough_money(ricci) is False
```

Comment:
Assertions of booleans do not require `is bool`. You can just do:
`assert not movies.enough_money(ricci)`
`assert movies.enough_money(bob)`